src/components/RestaurantHeader/RestaurantHeader.tsx

```diff
@@ -10,67 +14,61 @@ type Props = {
 }
 
 const RestaurantHeader: React.FC<Props> = ({ restaurant }) => {
+  const theme = useTheme()
+  const styles = getStyles(theme)
```

Comment:
Possibly a performance topic to discuss but not implement, make this a hook and have the hook wrap the styles in useMemo.

Reply:
I almost did make these as a `useStyles` hook instead of a function call, and probably would in a real app.